src/main/java/teammates/common/util/HibernateUtil.java

```diff
@@ -44,4 +44,21 @@ public static SessionFactory getSessionFactory() {
     public static void shutdown() {
         getSessionFactory().close();
     }
+
+    public static SessionFactory setSessionFactoryForTesting(String username, String password, String url) {
+        Configuration cfg = new Configuration()
+                .setProperty("hibernate.dialect", "org.hibernate.dialect.PostgreSQLDialect")
+                .setProperty("hibernate.connection.driver_class", "org.postgresql.Driver")
+                .setProperty("hibernate.connection.username", username)
+                .setProperty("hibernate.connection.password", password)
+                .setProperty("hibernate.connection.url", url)
+                .setProperty("hibernate.hbm2ddl.auto", "validate")
+                .setProperty("show_sql", "true")
+                .addPackage("teammates.storage.sqlentity")
+                .addAnnotatedClass(Course.class)
+                .addAnnotatedClass(Instructor.class);
+        sessionFactory = cfg.buildSessionFactory();
```

Comment:
If possible can consider removing the duplication with the configuration in `getSessionFactory` since the only three different properties are `username`, `password` and `url`.

Reply:
Ok! Code refactored